commands/preview.js

```diff
@@ -17,32 +17,31 @@
  * @api commands
  */
 
-var qs = require('querystring');
+import qs from 'querystring'
 
-exports.command = function(url, bundle, callback) {
-    var browser = this;
+exports.command = function(url, bundle = 'https://localhost:8443/loader.js', callback) {
```

Comment:
Now that you're using es6 everything, do you still need to create the exports object? Or can you just say:

`export const command = function(...)`

Also, while you're at it, can you use an arrow function here?

Reply:
Addressed. Unfortunately we're unable to use an arrow function due to losing the ability to call 'argument.lengths'.